### ISSUE TITLE
Colorization

### ISSUE THREAD

**Issue by kendagriff** [dgrnbrg/spyscope#18]
Any interest in incorporating [puget](https://github.com/greglook/puget) for colorization, either as the out-of-the-box or optional behavior? I'd be happy to submit a pull request.

If not, any suggestions on how to make them work together?

**Comment by dgrnbrg** [comment]
Yes, I think that'd be a great enhancement. Does puget have integration
with vim and emacs for in-editor coloring? Either way, fipp would enable
removing some of the synchronization code that specifically handles
performance issues with pprint running in parallel.

**Comment by kendagriff** [comment]
https://github.com/dgrnbrg/spyscope/pull/19

**Comment by jaor** [comment]
hi.  is there any way of disabling colorization? i sometimes run my tests inside an emacs compilation buffer and i'd rather not have ansi colors there... thanks!

**Comment by kendagriff** [comment]
@jaor It's probably a simple PR, but you might look at this https://github.com/greglook/puget, too.

**Comment by jaor** [comment]
On Wed, Jun 07 2017, Kendall Buchanan wrote:

> @jaor It's probably a simple PR, but you might look at this
> https://github.com/greglook/puget, too.

thanks kendall.  i looked around, but since it's spyscope whose calling
puget under the rug, i didn't see an easy way of telling `#spy/p` to
pass printer configuration params in its calls to puget.  i guess i can
set somewhere in my `~.lein/profiles.clj` a binding for
`puget.printer/*options*`, but i was wondering whether there was
something quick on spyscope's side.

**Comment by dgrnbrg** [comment]
I think that setting the binding is your best bet. I was going to suggest adding a binding to spyscope, but since puget already has that, you should be set!

**Comment by jaor** [comment]
On Wed, Jun 07 2017, David Greenberg wrote:

> I think that setting the binding is your best bet. I was going to
> suggest adding a binding to spyscope, but since puget already has
> that, you should be set!

i still think a spyscope-specific setting would be a good idea, and
would also protect your users from surprises the day you decide to
switch from puget to something else (if that day ever arrives; but then
a switch has already happened, and it has surprised me for one).

fwiw, imho, etc.

thanks!
